Dockerfile-build

```diff
@@ -0,0 +1,77 @@
+# This is an image that should been able to generate Swift libsbp bindings.
```

Comment:
Does this need the standard header?

Reply:
I don't know, but I can add one....

Reply:
orion dockerfiles don't have headers, so I'm going to say "not needed" unless advised otherwise.

Reply:
added